examples/nidcpower_source_dc_voltage/teststand_fixture.py

```diff
@@ -50,32 +52,12 @@ def create_nidcpower_sessions(sequence_context: Any) -> None:
         pin_map_id = teststand_support.get_active_pin_map_id()
 
         pin_map_context = nims.session_management.PinMapContext(pin_map_id=pin_map_id, sites=None)
-        with session_management_client.reserve_sessions(
-            context=pin_map_context,
-            instrument_type_id=nims.session_management.INSTRUMENT_TYPE_NI_DCPOWER,
-            # This code module sets up the sessions, so error immediately if they are in use.
```

Comment:
Don't delete the comment explaining `timeout=0`.

Reply:
Updated the comment for `timeout=0`